### PR TITLE
Fixes l6 saw having ridiculous ammo caliber

### PR DESCRIPTION
**Pull request by Bawhoppen**
I made L6 saw use a ridiculous 1.95x129mm ammo (basically a pin covered in gunpowder) because it needed to be changed from 5.56 flavor-wise, but I didn't really know what the numbers meant at all. 
Now I know a little more, and I'm so sorry. This is a more realistic (yet still fictional) ammo caliber.

### CODE REVIEW

**Comment by plapatin** [comment]
1.95x129mm ammo reserved for a clown gun that shoots thumbtacks when

**Comment by MrDoomBringer** [comment]
image
tfw removing meme calibers

**Comment by JStheguy** [comment]
It's 2558, if the syndies can have light sabers and magic hack-anything cards, then they can have their machine guns be chambered in Space H&K's latest contrarian meme cartridge.

File: code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -558,29 +558,29 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	include_modes = list(/datum/game_mode/nuclear)
 
 /datum/uplink_item/ammo/machinegun/basic
-	name = "1.95x129mm Box Magazine"
```
[the remark this answers is not in the document]
Comment (Bawhoppen) [reply code/modules/uplink/uplink_items.dm]: Really? I searched for it and I didn't see anything else. I'll have to check.

[the remark this answers is not in the document]
Comment (Bawhoppen) [reply code/modules/uplink/uplink_items.dm]: Looks like it did. Thanks for the heads up!

**Pull request merged**